readthedocs/core/apps.py

```diff
@@ -0,0 +1,12 @@
+from django.apps import AppConfig
+
+
+class CoreAppConfig(AppConfig):
+    name = 'readthedocs.core'
+    verbose_name = 'Core'
+
+    def ready(self):
+        if hasattr(self, 'already_run'):
+            return
+        self.already_run = True
```

Comment:
Yea, I was wondering about that myself -- it's in all of our `ready` calls, but seems unnecessary.